comp/dogstatsd/server/rc.go

```diff
@@ -0,0 +1,105 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package server implements a component to run the dogstatsd server
+package server
+
+import (
+	"encoding/json"
+	"slices"
+
+	"github.com/DataDog/datadog-agent/pkg/remoteconfig/state"
+)
+
+type statsdBlocklistUpdate struct {
+	BlockedMetrics blockedMetrics `json:"blocked_metrics"`
+}
+
+type blockedMetrics struct {
+	ByName byName `json:"by_name"`
+}
+
+type byName struct {
+	ConfigurationID int           `json:"configuration_id"`
+	Metrics         []metricEntry `json:"values"`
+}
+
+type metricEntry struct {
+	Name string `json:"metric_name"`
+}
+
+func (s *server) onBlocklistUpdateCallback(updates map[string]state.RawConfig, applyStateCallback func(string, state.ApplyStatus)) {
+	s.log.Debug("onBlocklistUpdateCallback received updates:", len(updates))
+
+	if len(updates) == 0 {
+		return
+	}
+
+	var blocklistUpdates []blockedMetrics
+
+	// unmarshal all the configurations received from
+	// the RC platform
+	// ------------------------
+
+	if len(updates) > 0 {
+		// unmarshal all configs that can be unmarshalled
+		for configPath, v := range updates {
+			s.log.Info("received:", string(v.Config))
```

Comment:
Would it make sense to lower this do debug level?

```suggestion
			s.log.Debugf("received: %q", string(v.Config))
```